packages/devtools_app/lib/src/example/conditional_screen.dart

```diff
@@ -93,9 +93,9 @@ class _ExampleConditionalScreenBodyState
 
   @override
   bool shouldLoadOfflineData() {
-    return offlineMode &&
-        offlineDataJson.isNotEmpty &&
-        offlineDataJson[ExampleConditionalScreen.id] != null;
+    return offlineController.offlineMode &&
```

Comment:
nit: seems like this could be a helper on offlineController.

Reply:
done